content/kapp-controller/docs/latest/air-gapped-workflow.md

```diff
@@ -0,0 +1,106 @@
+[`Option 2`](/imgpkg/docs/latest/air-gapped-workflow/#option-2-with-intermediate-tarball) can be found at the attached links. 
+
+A summary of steps for relocating a PackageRepository bundle or image to an air-gapped environment are documented for both options below:
+
+For `Option 1`: 
+* Get to a location that can access both registries. If there is no such location, you will have to use `Option 2` steps.
+* [Authenticate](/imgpkg/docs/latest/auth.md) with both source and destination registries
+* Run `imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-repo registry.corp.com/apps/simple-app-bundle` (Replace `-b` with `-i` for images)
+
+For `Option 2`:
+* Get to a location that can access source registry
+* [Authenticate] with the source registry(/imgpkg/docs/latest/auth.md)
+* Run `imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-tar /tmp/my-image.tar` (Replace `-b` with `-i` for images)
+
+Once you have your bundle or image in a location where you can access the registry in your air-gapped environment, move the bundle or image to its 
+final location:
+
+For `Option 1`: 
+* Run `imgpkg copy -b registry.corp.com/apps/simple-app-bundle --to-repo final-registry.corp.xyz/apps/simple-app-bundle` (Replace `-b` with `-i` for images)
+
+For `Option 2`:
+* `imgpkg copy --tar /tmp/my-image.tar --to-repo final-registry.corp.xyz/apps/simple-app-bundle` (Replace `-b` with `-i` for images)
+
+## Use Relocated Bundle or Image with PackageRepository
+
+Once you have relocated the PackageRepository bundle or image into the destination OCI registry in your air-gapped environment, you can 
+now reference the relocated bundle or image in the PackageRepository definition:
+
+```yaml
+---
+apiVersion: install.package.carvel.dev/v1alpha1
+kind: PackageRepository
+metadata:
+  name: air-gapped-repo
+spec:
+  fetch:
+    imgpkgBundle:
+      image: final-registry.corp.xyz/apps/simple-app-bundle
+```
+
+In the event your PackageRepository needs authentication to pull the bundle or image, you can specify credentials via a `secretRef`:
+
+```yaml
+---
+apiVersion: install.package.carvel.dev/v1alpha1
+kind: PackageRepository
+metadata:
+  name: simple-package-repository
+spec:
+  fetch:
+    imgpkgBundle:
+      image: final-registry.corp.xyz/apps/simple-app-bundle
+      secretRef:
+        name: my-registry-creds
+```
+
+The secret for the `secretRef` property must be created in the `kapp-controller` namespace for the PackageRepository to use this secret, 
+but this may change in the future. Supported secret keys are documented [here](config#imgpkgbundle-authentication).
+
+After applying the PackageRepository definition above to your Kubernetes cluster, you will be able to check that the PackageRepository and 
+its associated Packages were successfully deployed by checking the PackageRepository status:
+
+```bash
+$ kubectl get pkgr simple-package-repository
```

Comment:
pkgr -> packagerepository to avoid using shorthands (not everyone knows what it means).